components/server/src/auth/authenticator.ts

```diff
@@ -103,17 +104,20 @@ export class Authenticator {
         const host: string = req.query.host;
         const authProvider = host && await this.getAuthProviderForHost(host);
         if (!host || !authProvider) {
+            increaseLoginCounter("failed", "unkown")
```

Comment:
I'm tempted to call this an invalid request. No regular call should end up here, i.e. we can remove it. Sure, one can prepare an URL to reach this path, but I think that should be treated a code 400.

Reply:
Alright, makes sense :)